vendor/google.golang.org/grpc/server.go

```diff
@@ -163,11 +197,25 @@ func RPCDecompressor(dc Decompressor) ServerOption {
 	}
 }
 
-// MaxMsgSize returns a ServerOption to set the max message size in bytes for inbound mesages.
-// If this is not set, gRPC uses the default 4MB.
+// MaxMsgSize returns a ServerOption to set the max message size in bytes the server can receive.
+// If this is not set, gRPC uses the default limit. Deprecated: use MaxRecvMsgSize instead.
 func MaxMsgSize(m int) ServerOption {
```

Comment:
@dperny Just saw this was marked deprecated; perhaps you can update https://github.com/docker/swarmkit/blob/8aa9c33bcdff9ea38fc79e0b1d054199917513f3/manager/manager.go#L272, which is using the deprecated function